recommendations/test_sms_parser.py

```diff
@@ -155,7 +151,9 @@ def test_parse_is_view_list_when_invalid(self):
         self.assertRaises(ValueError, SmsParser.parse, message, self.phone, session)
 
     def test_parse_is_invite(self):
-        invite_number = self.faker.phone_number()
+        # Unfortunately, fakers phone numbers have extensions, which we don't support for now
+        # invite_number = self.faker.phone_number()
+        invite_number = '999-555-3322'
```

Comment:
Annoyingly Faker uses phone numbers with extensions, and there's no way to turn it off. I don't think we should support phone numbers with extensions as they're likely not cell phones, so I just hard-coded a number for now.

Reply:
Ok. We are using faker phone numbers other places. Probably should just set this invite number and a sender phone number in the setup so we can reuse them in other tests. I'll make an issue for it.